spec/puppetserver/ca/action/import_spec.rb

```diff
@@ -249,12 +249,21 @@
                                   'certname' => 'foocert',
                                   'subject-alt-names' => '' })
         expect(exit_code).to eq(0)
-        expect(File.exist?(File.join(tmpdir, 'ca', 'ca_crl.pem'))).to be true
-        expect(File.exist?(File.join(tmpdir, 'ca', 'ca_key.pem'))).to be true
-        expect(File.exist?(File.join(tmpdir, 'ca', 'ca_crt.pem'))).to be true
-        expect(File.exist?(File.join(tmpdir, 'ssl', 'certs', 'foocert.pem'))).to be true
-        expect(File.exist?(File.join(tmpdir, 'ssl', 'private_keys', 'foocert.pem'))).to be true
-        expect(File.exist?(File.join(tmpdir, 'ssl', 'public_keys', 'foocert.pem'))).to be true
+        created_correctly = ->(*args) do
+          perms = args.pop
+          file = File.join(tmpdir, *args)
+          File.exists?(file) &&
+            File.stat(file).mode.to_s(8)[-3..-1] == perms
+        end
```

Comment:
However, because this is in two places, maybe a better option would be to define it as a spec helper in another file. We already have a directory for test utilities here https://github.com/puppetlabs/puppetserver-ca-cli/tree/master/spec/utils. Is there a reason you didn't go this route?

Reply:
Yeah, that's a good point. I had originally written lambdas to wrap the tmpdir and directory, ie
```rb
cadir = ->(file) { File.join(tmpdir, 'ca', file) }
File.exists?(cadir['ca_crt.pem'])
```
And then kept the tmpdir wrapping when I turned it into more of a test assertion helper. I'll pull all these assertions into their own test helper.

Reply:
Ah yeah, that makes sense. Sounds good, probably worth it, especially if you can remove even more duplication than just the lambda.